src/coinify/coinify.js

```diff
@@ -400,8 +427,17 @@ Coinify.prototype.request = function (method, endpoint, data) {
     options.headers['Authorization'] = 'Bearer ' + this._access_token;
   }
 
```

Comment:
I copied it partially to make this code work independently and partially because Coinify's API is more pure than our own, so we need less complex code. I do think we can move some of this into a common library later on.